src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -929,7 +935,10 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         },
         [](auto args) {
             return fuzzyToInt(args.value, 5);
-        });
+        },
+        true,
+        "A value of 5s means if there's a 5s break inbetween messages, we will "
```

Comment:
"inbetween" is wrong. IMO this should be:
```suggestion
        "A value of 5s means if there's a 5s break between messages, we will "
```